docs/deployment.md

```diff
@@ -0,0 +1,56 @@
+# Deployment
+
+*This section is under construction*
+
+We will soon provide Docker images for individual Jaeger components,
+as well as orchestration configurations, for example to run them on Kubernetes.
+
+## Agent
+
+Jaeger client libraries expect `jaeger-agent` process to run locally on each host.
+
+### Discovery System Integration
+
+The agents can connect point to point to a single collector,
+or use a discovery system to load balance across several collectors.
+We use [go-kit](https://github.com/go-kit/kit) to support a number of different
+discovery systems, but currently blocked by [Issue 492](https://github.com/go-kit/kit/pull/492).
+
+TODO: [#134: allow agents to connect to multiple collectors via static list](https://github.com/uber/jaeger/issues/134)
+
+## Collectors
+
+Collectors require almost no configuration except for the location of Cassandra cluster.
+All configuration parameters can be provided via command line options.
+
+```go
+go run ./cmd/collector/main.go -h
+```
+
+To point collectors to Cassandra cluster, specify `-cassandra.keyspace` and `-cassandra.servers`
```

Comment:
We have a little bug where if you don't pass in -cassandra.connections-per-host, the collector will fatal out because there's no default.

Ditto for query.